README.md

```diff
@@ -153,11 +153,17 @@ Follow these [instructions](https://hatch.pypa.io/latest/install/) to install it
 ### Tests
 
 ```sh
-# Run all tests
-hatch run test
+# Run all tests for current Python version.
+hatch test
```

Comment:
Just to confirm, this only runs the unit tests correct? (i.e. doesn't include the integration tests)

Reply:
Correct! By default the "args" to `hatch test` is the tests folder (https://hatch.pypa.io/1.12/config/internal/testing/#default-arguments). We need to explicitly pass `hatch test integration_tests` for integration tests.